extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/supervisor/KafkaSupervisor.java

```diff
@@ -635,7 +649,7 @@ private void addDiscoveredTaskToPendingCompletionTaskGroups(
     }
 
     log.info("Creating new pending completion task group for discovered task [%s]", taskId);
-    TaskGroup newTaskGroup = new TaskGroup(startingPartitions);
+    TaskGroup newTaskGroup = new TaskGroup(startingPartitions, Optional.<DateTime>absent());
```

Comment:
yeah that's correct, for publishing tasks since there's no way that it can ingest any more data, the minimum message time isn't necessary.